docs/reason-demo/stdlib/caml_c_ffi.js

```diff
@@ -17,7 +17,7 @@
 // along with this program; if not, write to the Free Software
 // Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 //  Copyright (c) 2015 Bloomberg LP. All rights reserved. 
-// Hongbo Zhang (hzhang295@bloomberg.net)              
+// Hongbo Zhang (hzhang295@bucklescript.net)              
```

Comment:
Are you sure this is correct, or did you go a bit too far with the "replace all"?